app/src/data/data.ts

```diff
@@ -0,0 +1,75 @@
+export const DEFAULT_FONT = 'Noto Sans Regular';
+
+export const EXAMPLES = [
+    {
+        name: 'Font "Barlow Regular"',
```

Comment:
Nitpick: it might be a bit confusing to the reader to call a single-font output a `Font` and a multi-font output a `Font stack`. 

From the [MapLibre docs](https://maplibre.org/maplibre-style-spec/glyphs/) any glyph set is called a "fontstack" regardless of whether it contains a single font or multiple fonts.